<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldFirstPlayer</class>
 <widget class="QMainWindow" name="FieldFirstPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">FieldFirstPlayer {background-image: url(:/image/image/sea.jpg)}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/image/back.png</normaloff>:/image/image/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="nicknameEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44);
</string>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>730</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string>AUTO</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>730</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>503</width>
      <height>503</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="firstShipButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>200</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string> - 4</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/image/firstShip.png</normaloff>:/image/image/firstShip.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="secondShipButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>320</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string> - 3</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/image/secondShip.png</normaloff>:/image/image/secondShip.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="thirdShipButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>440</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string> - 2</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/image/thirdShip.png</normaloff>:/image/image/thirdShip.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="forthShipButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>560</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 69, 44)</string>
    </property>
    <property name="text">
     <string> - 1</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/image/image/forthShip.png</normaloff>:/image/image/forthShip.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
